bot.py

```diff
@@ -68,7 +70,38 @@ async def on_message(message):
             line = classes_offered.loc[classes_offered['Class'] == class_str]
             
             if len(line) == 0:
-                await message.channel.send(class_str + ': Could not find this class. It is likely not offered in FA 2020.\n')
+                # check if page exists in course explorer
+                if requests.get("https://courses.illinois.edu/schedule/terms/"+course[0].upper()+"/"+course[1]).status_code == 200:
+                    # get page & parse w BS4
+                    page = requests.get("https://courses.illinois.edu/schedule/terms/"+course[0].upper()+"/"+course[1])
+                    soup = BeautifulSoup(page.content, 'html.parser')
+                    all_a_tags = soup.find_all('a')
+                    # get all terms that the class has been offered
+                    terms_offered = []
+                    for a in all_a_tags:
```

Comment:
If we're going to pick the first element anyways, why iterate through all of them?